packages/docs/src/routes/demo/cookbook/glob-import/index.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  type Component,
+  component$,
+  useSignal,
+  useTask$,
+} from '@builder.io/qwik';
+import { isDev } from '@builder.io/qwik/build';
+
+const metaGlobComponents: any = import.meta.glob(
```

Comment:
can we use a better type other then any?

Reply:
@gioboa I spent another hour on this but typescript is lost, probably because the `isDev` value depends on whether we are in development or not, and also because import.meta.glob will be of type 
- `Record<string, () => Promise<unknown>>` with `eager: false`